src/booleans.js

```diff
@@ -1,61 +1,65 @@
-function negate(a) {
-  // your code here
+const negate = (a) => {
+  return !(a);
 };
 
-function both(a, b) {
-  // your code here
+const both = (a, b) => {
+  return (a && b);
 };
 
-function either(a, b) {
-  // your code here
+const either = (a, b) => {
+  return (a || b);
 };
 
-function none(a, b) {
-  // your code here
+const none = (a, b) => {
+  return !(a || b)
 };
 
-function one(a, b) {
-  // your code here
+const one = (a, b) => {
+  return ((a && !b) || (!a && b));
 };
 
-function truthiness(a) {
-  // your code here
+const truthiness = (a) => {
+  if (a) {
+    return true
+  } else {
+    return false
+  }
```

Comment:
This works, but there is a cleaner way. Remember all values in JS are either truthy or falsy:
'dfdf' = truthy
true = true
0 = false
undefined = false

So we cannot just return. whatever is passed in, but what we can do is negate it to turn it into a boolean value with the `!` not operator: 
if a = 'dfdf' (truthy), then !a == false
true = true (truthy), then !a == false
0 = false (falsy), then !a == true

So by negating _that_ value we can work out the truthiness of the original value, eg
if a = 'dfdf' (truthy), then !a == false so !!a == true

So for this function we could just return:
```suggestion
  return !!a;
```

Hope that makes sense!